sdk/python/pulumi_kubernetes/helm/v2/helm.py

```diff
@@ -295,13 +295,22 @@ def __init__(self,
         self.path = path
 
 
-def _run_helm_cmd(cmd: List[Union[str, bytes]]):
+def _run_helm_cmd(all_config: Tuple[List[Union[str, bytes]], pulumi.Output]):
```

Comment:
Just wondering - Is the type annotation `pulumi.Output` here actually correct?  I would have assumed that since this is called via an `apply` the values in the tuple were all actual values instead of `pulumi.Output`'s?  I am guessing if we get this wrong it's not a big problem because the type hints aren't enforced, just trying to understand the system we have.

Reply:
Hmm, I'm not actually sure, but I suspect you're correct.

Reply:
Updated the type hints.